bitcoin/src/blockdata/locktime/absolute.rs

```diff
@@ -459,6 +459,9 @@ impl Height {
 impl fmt::Display for Height {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result { fmt::Display::fmt(&self.0, f) }
 }
+impl Default for Height {
+    fn default() -> Self { Height::from_consensus(0).expect("Failed to create a default Height") }
```

Comment:
In e6957d2259b34323e5bed7d33651164d3e8626ac:

You can just directly construct the value and avoid the panic branch.